tests/serverpod_test_server/test_e2e/basic_types_test.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/test.dart';
+
+import '../test/serialization_legacy_test.dart' show createByteData;
+
+void main() {
+  var client = Client(serverUrl);
+
+  test(
+      'Given the test server, when the global int is set to 10 and 1 is added, then 11 is returned',
+      () async {
+    await client.simple.setGlobalInt(10);
+    await client.simple.addToGlobalInt();
+    var value = await client.simple.getGlobalInt();
+    expect(value, equals(11));
+  });
```

Comment:
Question: This doesn't seem like a types test, what are we testing here? I can see that it was there in the old test suite, I think we can simply move this one back and deal with it another day.

Reply:
Right. I moved this back out.